core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessorState.java

```diff
@@ -70,6 +71,7 @@
     private final NanosecondClock nanosecondClock;
     private final int floatScale;
     private final int doubleScale;
+    private final SCSequence tempConsumeSequence = new SCSequence();
```

Comment:
could we call this `eventSubSeq` ?

it isn't temporary and it would be good to settle on `pub/sub` nomenclature for sequences for consistency

Reply:
ok